terraform/globals.tf

```diff
@@ -100,5 +100,11 @@ variable "microbosh_static_private_ip" {
 /* Note: This is overridden in prod.tfvars and staging.tfvars to allow world access */
 variable "web_access_cidrs" {
   description = "CSV of CIDR addresses for which we allow web access"
-  default     = "80.194.77.90/32,80.194.77.100/32,85.133.67.244/32"
+  default     = ""
+}
+
+/* Note: This is overridden in prod.tfvars to allow access only in prod*/
```

Comment:
'to allow access only in prod' ? Also, web_access_cirds now acts as extra IPs to enable - but is in a different format/way to the new method of this PR